src/components/EventCard.jsx

```diff
@@ -59,12 +59,25 @@ const EventCard = (props) => {
     >
       <div className="flex justify-between">
         <div className="flex justify-start">
-          <DateDisplayComponent
-            date={props.event.date}
-            version={props.version ?? "Primary"}
-          />
+          {props.dateDisplay ? (
+            <DateDisplayComponent
+              date={props.event.date}
+              version={props.version ?? "Primary"}
+            />
+          ) : (
+            <></>
+          )}
           <div className="flex-column flex text-xl">
-            <Text text={props.event.title} type="subheading" />
+            <div className="flex">
```

Comment:
Is there a reason you didn't use the flowbite component? https://flowbite-react.com/badges